operator/src/main/java/oracle/kubernetes/operator/steps/ManagedServerUpIteratorStep.java

```diff
@@ -102,6 +119,27 @@ private Packet createPacketForServer(Packet packet, ServerStartupInfo ssi) {
     return p;
   }
 
+  private Map<String, StartClusteredServersStepFactory> getStartClusteredServersStepFactories(
+      Collection<ServerStartupInfo> startupInfos,
+      Packet packet) {
+    DomainPresenceInfo info = packet.getSpi(DomainPresenceInfo.class);
+    Domain domain = info.getDomain();
+
+    Map<String, StartClusteredServersStepFactory> factories = new HashMap<>();
+    startupInfos.stream()
+        .filter(ssi -> isServerInCluster(ssi))
```

Comment:
If you're using Intellij, it should have suggested that you change this to

.filter(this::isServerInCluster)

since there is no need to repeat the variable ssi here

Reply:
done